common/djangoapps/course_modes/models.py

```diff
@@ -553,8 +553,8 @@ def to_tuple(self):
         )
 
     def __unicode__(self):
-        return u"{} : {}, min={}, prices={}".format(
-            self.course_id.to_deprecated_string(), self.mode_slug, self.min_price, self.suggested_prices
+        return u"{} : {}, min={}".format(
+            self.course_id.to_deprecated_string(), self.mode_slug, self.min_price   # pylint: disable=no-member
```

Comment:
`to_deprecated_string()` is no longer necessary -- you can just use `self.course_id` here and it will be automatically typecast to `unicode`